package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "sync-fork",
+  "version": "0.0.0",
```

```suggestion
  "version": "0.1.0",
```

suggestion for semantic versioning-compilant